Sources/SPMBuildCore/PluginInvocation.swift

```diff
@@ -631,525 +712,170 @@ public extension PluginInvocationDelegate {
     }
 }
 
```

Comment:
The diffs in the following section aren't easy to follow, but essentially the redeclaration of the serializable struct is removed (since it's now shared with the PackagePlugin module via PluginMessages.swift) but is instead replaced with extensions that instantiate the client-facing structure from the Codable structure.